.github/workflows/tests.yml

```diff
@@ -102,9 +110,17 @@ jobs:
       - name: Install PostgreSQL on the runner
         run: brew install postgresql@13
 
+      - name: Install PostGIS extension on the runner
+        run: brew install postgis
+        if: ${{ matrix.postgis == 'yes' }}
+
       - name: Start PostgreSQL service for test
         run: brew services start postgresql
 
+      - name: Enable PostGIS extension
```

Comment:
In other tests, setting prerequisites is done by the unit test, using a fixture. See hstore for an example.

Not sure it's better in absolute terms. If it fails, tests are skipped instead of failing, which is not really optimal. However it keeps the pipeline simpler.

I'll take a look myself at whether it can be improved.

Reply:
I think now that macOS postgis test is gone it's not needed anymore